Source/WTF/wtf/WTFConfig.h

```diff
@@ -83,7 +83,7 @@ struct Config {
     bool isThreadSuspendResumeSignalConfigured;
     int sigThreadSuspendResume;
 #endif
-#if OS(UNIX)
+#if OS(UNIX) || OS(WINDOWS)
```

Comment:
@Constellation is there any `OS` that isn't `UNIX` or `WINDOWS`? Only one I'm not sure about is `FUSCIA`.

If not then the guard can just be dropped.

Reply:
https://github.com/WebKit/WebKit/blob/main/Source/WTF/wtf/PlatformOS.h#L121-L134

`FUCHSIA` gets bundled as Unix-like, so I think we could drop this guard.